rotkehlchen/tests/unit/decoders/test_cowswap.py

```diff
@@ -0,0 +1,366 @@
+import pytest
+from rotkehlchen.accounting.structures.balance import Balance
+from rotkehlchen.accounting.structures.base import HistoryBaseEntry
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.assets.asset import Asset
+from rotkehlchen.chain.ethereum.modules.cowswap.constants import CPT_COWSWAP
+from rotkehlchen.chain.evm.decoding.constants import CPT_GAS
+from rotkehlchen.constants.assets import A_CRV, A_ETH, A_USDC, A_USDT, A_WBTC, A_WETH
+from rotkehlchen.fval import FVal
+from rotkehlchen.tests.utils.ethereum import get_decoded_events_of_transaction
+
+from rotkehlchen.types import Location, TimestampMS, deserialize_evm_tx_hash
+
+
+@pytest.mark.vcr()
```

Comment:
relevant tests fail. You need to change this to
```suggestion
@pytest.mark.vcr(filter_query_parameters=['apikey'])
```

Same thing for places where you use var

Reply:
You did not seem to do the change that Isaac recommended. Why?